app/assets/javascripts/listings/templates/property-card/_rental-stats-row.html.slim

```diff
@@ -1,40 +1,40 @@
 tr ng-repeat-start="unit_summary in ::summary" ng-if="::unit_summary.minRentalMinIncome || unit_summary.maxRentalMinIncome"
   / Dollar based rent and min income
-  td.is-subtitled data-th="Unit Type"
+  td.is-subtitled data-th="{{'t.unit_type' | translate}}"
     | {{::unit_summary.unitType}}
-  td.is-subtitled data-th="Min. Income"
+  td.is-subtitled data-th="{{'t.min_income' | translate}}"
```

Comment:
might be nice to update this variable name to make it clear that it's the shortened mobile view title?

e.g. min_income_short?